app/views/coronavirus/pages/prepare.html.erb

```diff
@@ -5,78 +5,68 @@
+      <%= t("coronavirus.pages.github.approve_sub_heading") %>
+    </p>
+    <%= render "govuk_publishing_components/components/heading", {
+      text: t("coronavirus.pages.draft.heading"),
+      padding: true
+    } %>
+    <%= form_with(url: coronavirus_page_path, method: "put", local: true) do %>
+      <%= render "govuk_publishing_components/components/button", {
+          text: t("coronavirus.pages.draft.button_text"),
+          margin_bottom: true
       } %>
-      <p class="govuk-body">
-        <%= t("coronavirus.pages.github.update_sub_heading_html", github_url: @page.github_url) %>
-      </p>
-    </div>
-    <div class="govuk-!-padding-bottom-1">
-      <%= render "govuk_publishing_components/components/heading", {
-        text: t("coronavirus.pages.github.review_heading"),
-        padding: true
+    <% end %>
+    <%= render "govuk_publishing_components/components/heading", {
+      text: t("coronavirus.pages.preview.heading"),
+      padding: true
+    } %>
+      <%= render "govuk_publishing_components/components/button", {
+        text: t("coronavirus.pages.preview.button_text"),
+        href: draft_govuk_url(@page.base_path),
+        target: "_blank",
+        secondary: true,
+        margin_bottom: true,
       } %>
-      <p class="govuk-body">
-        <%= t("coronavirus.pages.github.approve_sub_heading") %>
-      </p>
-    </div>
-    <div class="govuk-!-padding-bottom-6">
-      <%= render "govuk_publishing_components/components/heading", {
-        text: t("coronavirus.pages.draft.heading"),
-        padding: true
+    <%= render "govuk_publishing_components/components/heading", {
+      text: t("coronavirus.pages.publish.heading"),
+      padding: true
+    } %>
+    <%= form_with(url: publish_coronavirus_page_path(params[:slug])) do %>
+      <%= render "govuk_publishing_components/components/radio", {
+        name: "update-type",
+        heading: t("coronavirus.pages.publish.radio_heading"),
+        heading_size: "s",
+        small: true,
+        items: [
+          {
+            value: "minor",
+            text: t("coronavirus.pages.publish.minor_type"),
+            checked: true
+          },
+          {
+            value: "major",
+            text: t("coronavirus.pages.publish.major_type")
+          }
+        ]
       } %>
-      <%= form_with(url: coronavirus_page_path, method: "put", local: true) do %>
-        <%= render "govuk_publishing_components/components/button", {
-            text: t("coronavirus.pages.draft.button_text"),
-            margin_bottom: false
-        } %>
-      <% end %>
-    </div>
-    <div class="govuk-!-padding-bottom-2">
-      <%= render "govuk_publishing_components/components/heading", {
-        text: t("coronavirus.pages.preview.heading"),
-        padding: true
+      <%= render "govuk_publishing_components/components/button", {
+          text: t("coronavirus.pages.publish.button_text"),
+          margin_bottom: true,
```

Comment:
This looks a good iteration - does it change visually at all?

Reply:
I think it adds an extra line between sections. Barely noticeable in my humble opinion.